README.md

```diff
@@ -124,6 +124,47 @@ does have the side effect that, if an internal username is connected to multiple
 GitHub usernames, as demonstrated above, all of those GitHub usernames will be
 made admins of the org.
 
+### [[applications]]
+
+Each `[[applications]]` section represents one org-level application
+installation. These entries are used to ensure that no applications get added to
+or removed from the org unexpectedly. You can also ensure that they only have
+the permissions you are expecting, and receive the events you are expecting.
+
+Similar to the `[[teams]]` above, applications are specified as an [array of
+tables]. In addition, the applications use a sub-table for the permissions,
+making a single application specification look like the following:
+
+```toml
+[[applications]]
+appId = 42
+appSlug = "infinite-improbability-drive"
+repositorySelection = "selected"
```

Comment:
as a consumer of this library, where do I find the values to use for these?

Reply:
Hmmm yeah good question. The app id and slug I was only able to find after actually querying our org. I honestly don't know of a better way to find them, so that may have to be our recommendation for now: run the tool and plug the data into your config. Which gives me a great idea for a future feature: `orglinter --generate-config` It could query the org and produce a valid config file based on its current state.

Oh as for the `repositorySelection` I can document that "selected" and "all" are the only values. I think. I haven't been able to find solid documentation on that either. The REST API docs are terrible!